pixeldata/Cargo.toml

```
@@ -25,13 +25,14 @@ dicom-dictionary-std = { path = "../dictionary-std", version = "0.9" }
 snafu = "0.8"
 byteorder = "1.4.3"
 gdcm-rs = { version = "0.6", optional = true }
-rayon = { version = "1.5.0", optional = true }
+rayon = { version = ">= 1.5, < 1.11.0", optional = true }
+rayon-core = { version = "~1, < 1.13.0", optional = true }
 ndarray = { version = "0.16.1", optional = true }
 num-traits = "0.2.12"
 tracing = "0.1.34"
 
 [dependencies.image]
-version = "0.25.1"
+version = "<0.25.8"
```

I suspect that this could match versions behind `0.25.0` in theory.
```suggestion
version = ">=0.25.1 <0.25.8"
```